src/main/java/gregtech/common/blocks/BlockBrittleCharcoal.java

```diff
@@ -0,0 +1,44 @@
+package gregtech.common.blocks;
+
+import gregtech.api.GTValues;
+import net.minecraft.block.Block;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.MapColor;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.client.util.ITooltipFlag;
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class BlockBrittleCharcoal extends Block {
```

Comment:
I think this should have similar fire properties like Block of Coal. As it is strange that it does not burn down or spread fire given that it is basically Charcoal